e2e/catalogue/catalogue-test_1.md

```diff
@@ -0,0 +1,21 @@
+
+# Number
+
+1
+
+# Role
+
+Data manager of a network needs to check their network information.
```

Comment:
Role: Data manager ( of a network) , the rest is goal ( being strict is needed of we want to automate a some point )